contracts/Guess.sol

```diff
@@ -78,7 +78,9 @@ contract Guess {
         require(msg.sender == owner, "Only owner can call this function."); 
         _;
     }
-    
+
+    event AttemptWon(address _winnerAccount, uint _amount);
```

Comment:
can you make the winnerAccount as indexed.

Reply:
Done